apps/channel/client/channel_new_collaborator_view.coffee

```diff
@@ -22,7 +23,7 @@ module.exports = class newCollaboratorView extends Backbone.View
     @clear() unless $(e.target).closest('#metadata--collaborators').length
 
   addCollaborator: (e) ->
-    @collaborators._add $(e.target).data 'id'
+    @collaborators._add @results.get $(e.target).data 'id'
```

Comment:
In which case it'd be preferable to overwrite the model `sync` method. But carry on for now.